python/tvm/driver/tvmc/model.py

```diff
@@ -281,7 +320,9 @@ def export_package(
         if output_format == "mlf" and cross:
             raise TVMCException("Specifying the MLF output and a cross compiler is not supported.")
 
-        if output_format in ["so", "tar"]:
+        if use_vm:
```

Comment:
Can we check the type of `executor_factory` instead of having a `use_vm` input for this function?